SU2_CFD/src/solvers/CIncNSSolver.cpp

```diff
@@ -586,3 +599,275 @@ void CIncNSSolver::BC_ConjugateHeat_Interface(CGeometry *geometry, CSolver **sol
+   At this moment, the wall function is only available for adiabatic flows.
+   ---*/
+  su2double RefDensity,RefVel2; 
+  if ((config->GetRef_Inc_NonDim() == DIMENSIONAL) || (config->GetRef_Inc_NonDim() == INITIAL_VALUES)) {
+    RefDensity = Density_Inf;
+    RefVel2 = 0.0;
+    for (auto iDim = 0u; iDim < nDim; iDim++) 
+      RefVel2 += Velocity_Inf[iDim] * Velocity_Inf[iDim];
+  } else if (config->GetRef_Inc_NonDim() == REFERENCE_VALUES) {
+    RefDensity = config->GetInc_Density_Ref();
+    RefVel2 = config->GetInc_Velocity_Ref() * config->GetInc_Velocity_Ref();
+  }
+
+  unsigned long notConvergedCounter=0;
+
+  su2double grad_diff;
+  su2double U_Tau, Y_Plus;
+  const su2double Gas_Constant = config->GetGas_ConstantND();
+  const su2double Cp = (Gamma / Gamma_Minus_One) * Gas_Constant;
+  su2double Eddy_Visc;
+  constexpr unsigned short max_iter = 50;
+  const su2double tol = 1e-12; // 1e-10 is too large
+  const su2double relax = 0.5; 
+  
+    /*--- Compute the recovery factor ---*/
+  // Molecular (Laminar) Prandtl number (see Nichols & Nelson, nomenclature )
+  const su2double Recovery = pow(config->GetPrandtl_Lam(), (1.0/3.0));
+
+  /*--- Typical constants from boundary layer theory ---*/
+
+  const su2double kappa = 0.41; // put model constants in config file
+  const su2double B = 5.5;
+
+
+  for (auto iMarker = 0u; iMarker < config->GetnMarker_All(); iMarker++) {
+
+    if (!config->GetViscous_Wall(iMarker)) continue;
+
+    if ((config->GetMarker_All_KindBC(iMarker) == HEAT_FLUX) ||
+        (config->GetMarker_All_KindBC(iMarker) == ISOTHERMAL) ) {
+
+      /*--- Identify the boundary by string name ---*/
+
+      const auto Marker_Tag = config->GetMarker_All_TagBound(iMarker);
+
+      /*--- Jump to another BC if it is not wall function ---*/
+
+      if (config->GetWallFunction_Treatment(Marker_Tag) != STANDARD_WALL_FUNCTION)
+        continue;
+
+      /*--- Get the specified wall heat flux from config ---*/
+      // note that we can get the heat flux from the temperature gradient
+      su2double q_w = config->GetWall_HeatFlux(Marker_Tag);
+      // heat flux from temperature: q_w = h*(T_wall - T_fluid)
+
+      /*--- Loop over all of the vertices on this boundary marker ---*/
+
+      SU2_OMP_FOR_DYN(OMP_MIN_SIZE)
+      for (auto iVertex = 0u; iVertex < geometry->nVertex[iMarker]; iVertex++) {
+
+        const auto iPoint = geometry->vertex[iMarker][iVertex]->GetNode();
+        const auto Point_Normal = geometry->vertex[iMarker][iVertex]->GetNormal_Neighbor();
+
+        /*--- Check if the node belongs to the domain (i.e, not a halo node)
+         and the neighbor is not part of the physical boundary ---*/
+
+        if (!geometry->nodes->GetDomain(iPoint)) continue;
+
+          /*--- Get coordinates of the current vertex and nearest normal point ---*/
+
+          const auto Coord = geometry->nodes->GetCoord(iPoint);
+          const auto Coord_Normal = geometry->nodes->GetCoord(Point_Normal);
```

Comment:
Indentation of the code after the if, i.e. move code back a couple spaces please.
the reason I started using `continue` was to not indent the code so much to the right, we only use 2 space indents, after a few of those it was difficult to see what is what.